eraser-map.yaml

```diff
@@ -861,7 +861,6 @@ styles:
     icons:
         base: points
         texture: pois
-        interactive: true
```

Comment:
Yep, that was what I was thinking. I'd want to first look at how common
this actually is though, it fits into a more general semantic division
between styles and draw groups, which we may be able to improve in other
ways.

On Thu, Jan 14, 2016 at 1:25 PM, Nathaniel V. KELSO <
notifications@github.com> wrote:

> In eraser-map.yaml
> <https://github.com/tangrams/eraser-map/pull/136#discussion_r49763528>:
>
> > @@ -861,7 +861,6 @@ styles:
> >      icons:
> >          base: points
> >          texture: pois
> > -        interactive: true
>
> What about something like...
>
> icons:
>     base: points
>     texture: pois
>     draw
>         interactive: true
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tangrams/eraser-map/pull/136/files#r49763528>.
>